src/main/java/com/amazon/opendistroforelasticsearch/security/securityconf/impl/v6/NodesDnV6.java

```diff
@@ -0,0 +1,30 @@
+package com.amazon.opendistroforelasticsearch.security.securityconf.impl.v6;
```

Comment:
What is the difference between `NodesDnV6` and `NodesDnV7`?

Reply:
Though no difference in structure, the classes serve as a way to identify the config version. V7 versions when persisted to index includes meta fields while V6 versions don't.

I'll take another look to see if we can make it work with single class.

Reply:
Merged the classes.